e2e/src/helper/utils.ts

```diff
@@ -5,3 +5,10 @@ export function getRandom(): number {
 export function formatUserName(firstName: string, lastName: string): string {
   return `${firstName} ${lastName[0]}.`;
 }
+
+export function concatEducationDetailsString(education):string {
```

Comment:
1. create a model (interface for an education) to have a type for an argument.
2. 
```suggestion
export function concatEducationDetailsString(education): string {
```